Cargo.toml

```diff
@@ -51,9 +52,9 @@ tokio = { version = "1", features = ["io-util"] }
 rustc_tools_util = "0.3.0"
 
 [features]
-deny-warnings = ["clippy_lints/deny-warnings"]
-integration = ["tempfile"]
-internal = ["clippy_lints/internal", "tempfile"]
+deny-warnings = ["clippy_lints/deny-warnings", "clippy_internal_lints?/deny-warnings"]
+integration = ["dep:tempfile"]
+internal = ["dep:clippy_internal_lints", "dep:tempfile"]
```

Comment:
TIL about the `dep:` syntax and the `?` after a dependency :eyes:

Reply:
New in 1.60. The `dep:` part wasn't needed, but it's easier to read.